File: Tareas/Unidad2/a2253332351_practica02/17_VelocidadPromedio.java
import java.util.Scanner;

public class VelocidadPromedio {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double distancia, tiempo, velocidad;

          System.out.println("Ingrese la distancia recorrida en km: ");
          distancia = entrada.nextDouble();
          System.out.println("Ingrese el tiempo empleado en horas: ");
          tiempo = entrada.nextDouble();

          velocidad = distancia / tiempo;

          System.out.println("La velocidad promedio es: " + velocidad);
    }
}

File: Tareas/Unidad2/a2253332351_practica02/17_VelocidadPromedio.cpp
#include <iostream>
using namespace std;

int main() {
    double distancia;
    double tiempo;
    double velocidad;

    cout << "Ingrese la distancia recorrida en km: ";
    cin >> distancia;
    cout << "Ingrese el tiempo empleado en horas: ";
    cin >> tiempo;

    velocidad = distancia / tiempo;

    cout << "La velocidad promedio es: " << velocidad << endl;

    return 0;
}
